pyvista/core/filters/data_set.py

```diff
@@ -2588,8 +3068,8 @@ def plot_over_circular_arc_normal(dataset, center, resolution=None, normal=None,
 
         >>> from pyvista import examples
         >>> mesh = examples.load_uniform()
-        >>> normal = [mesh.bounds[0], mesh.bounds[2], mesh.bounds[5]]
-        >>> polar = [mesh.bounds[0], mesh.bounds[3], mesh.bounds[4]]
+        >>> normal = normal = [0, 0, 1]
+        >>> polar = [0, 9, 0]
         >>> angle = 90
         >>> center = [mesh.bounds[0], mesh.bounds[2], mesh.bounds[4]]
         >>> mesh.plot_over_circular_arc_normal(center, polar=polar, angle=angle)  # doctest:+SKIP
```

Comment:
No matplotlib plot generated either.

Reply:
We'd have to switch to `numpydoc` to get matplotlib plots rendered or copy the necessary code into a custom extension. 
 I've raised this issue with the sphinx docs folk.  I'd like to leave this for a later PR.

Reply:
Sure thing, I just flagged what I came across.